include/zephyr/mgmt/mcumgr/mgmt/mgmt.h

```diff
@@ -93,7 +93,7 @@ struct mgmt_group {
 	/** The numeric ID of this group. */
 	uint16_t mg_group_id;
 
-#if IS_ENABLED(CONFIG_MCUMGR_SMP_SUPPORT_ORIGINAL_PROTOCOL)
```

Comment:
this seems wrong, was told (a long time ago) that `defined()` would not work for documentation and would be excluded but `IS_ENABLED()` would always work in documentation @gmarull @kartben ?